cmake/FindGeode.cmake

```diff
@@ -73,7 +73,8 @@ if(Geode_gfsh_EXECUTABLE)
     # TODO error checking
   else()
     if(var MATCHES "([0-9]+\\.[0-9]+\\.[0-9]+)")
-      set(Geode_VERSION "${CMAKE_MATCH_1}")
+	  set(Geode_VERSION "${CMAKE_MATCH_1}")
+	  set(Geode_VERSION_FULL ${var})
```

Comment:
Unless I'm mistaken, nothing in this change depends on Geode_VERSION any more, so there should be no changes necessary in this file.  Please remove, we don't want to accidentally introduce any new ways to break the build.

Reply:
Good catch!